bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -340,6 +350,79 @@ def get_image_responses(self, sources, quality=None, pixel_format=None):
 
         return image_responses
 
+    def setup_logging_sources(self, camera_sources: List[str]):
+        """
+        Order .. RGB & then DEPTH_IN_RGB
+        DO NOT SUPPORT DEPTH and RGB_IN_DEPTH.
+        """
+        source_list = []  # type: List[str]
+
+        if not camera_sources:
+            print("Empty list passed in logger camera sources")
+
+        # Verification checks
+        for camera_source in camera_sources:
+            if (
+                "_depth" in camera_source
+                and "_depth_in_visual_frame" not in camera_source
+                and "_depth_in_hand_color_frame" not in camera_source
+            ):
+                print(
+                    f"Invalid source name {camera_source}. Camera source should either be rgb or depth aligned in rgb"
```

Comment:
why? why don't we just support all camera_source that someone might specify?

Reply:
done.